components/x-podcast-launchers/src/generate-app-links.js

```diff
@@ -0,0 +1,13 @@
+import appLinksConfig from './config/app-links';
+
+export default function generateAppLinks(rssUrl) {
+	return appLinksConfig.map(data => {
+		const url = data.includeProtocol
+					? rssUrl
+					: rssUrl.replace(/^https?:\/\//, '');
+
+		return Object.assign({}, data , {
+			url: data.template.replace(/{url}/, url)
+		})
+	});
+}
```

Comment:
I think this file needs Prettier run over it

Reply:
👍